TinyConsole/TinyConsoleController.swift

```diff
@@ -14,9 +14,20 @@ open class TinyConsoleController: UIViewController {
     ///
     /// - collapsed: the console is hidden
     /// - expanded: the console is shown
-    enum WindowMode {
-        case collapsed
+    public enum WindowMode: Int, CustomStringConvertible {
+        case collapsed = 0
         case expanded
+        
+        public var description: String {
```

Comment:
is it only for debugging or why exactly is this here? there is not even one call to description.